client/my-sites/site-settings/publishing-tools/index.jsx

```diff
@@ -105,32 +113,65 @@ class PublishingTools extends Component {
 		const formPending = this.isFormPending();
 
 		return (
-			<FormFieldset>
-				<SupportInfo
-					text={ translate(
-						'Allows you to publish new posts by sending an email to a special address.'
-					) }
-					link={
-						isAtomic
-							? localizeUrl( 'https://wordpress.com/support/post-by-email/' )
-							: 'https://jetpack.com/support/post-by-email/'
-					}
-					privacyLink={ ! isAtomic }
-				/>
-				<JetpackModuleToggle
-					siteId={ selectedSiteId }
-					moduleSlug="post-by-email"
-					label={ translate( 'Publish posts by sending an email' ) }
-					disabled={ formPending || moduleUnavailable }
-				/>
+			<>
+				<FormFieldset>
+					<SupportInfo
+						text={ translate(
+							'Allows you to publish new posts by sending an email to a special address.'
+						) }
+						link={
+							isAtomic
+								? localizeUrl( 'https://wordpress.com/support/post-by-email/' )
+								: 'https://jetpack.com/support/post-by-email/'
+						}
+						privacyLink={ ! isAtomic }
+					/>
+					<JetpackModuleToggle
+						siteId={ selectedSiteId }
+						moduleSlug="post-by-email"
+						label={ translate( 'Publish posts by sending an email' ) }
+						disabled={ formPending || moduleUnavailable }
+					/>
+
+					{ this.renderPostByEmailSettings() }
+				</FormFieldset>
+				<hr />
+			</>
```

Comment:
```suggestion
			<CompactCard className="site-settings__module-settings">
				<QueryJetpackConnection siteId={ selectedSiteId } />

				<FormFieldset>
					<SupportInfo
						text={ translate(
							'Allows you to publish new posts by sending an email to a special address.'
						) }
						link={
							isAtomic
								? localizeUrl( 'https://wordpress.com/support/post-by-email/' )
								: 'https://jetpack.com/support/post-by-email/'
						}
						privacyLink={ ! isAtomic }
					/>
					<JetpackModuleToggle
						siteId={ selectedSiteId }
						moduleSlug="post-by-email"
						label={ translate( 'Publish posts by sending an email' ) }
						disabled={ formPending || moduleUnavailable }
					/>

					{ this.renderPostByEmailSettings() }
				</FormFieldset>
			</CompactCard>
```
Compared to the other settings modules on this page, the `<hr>` color looks slightly off. I think it'd be better to remove the `<hr>` and wrap these components in `CompactCard`